### PR TITLE
Update doc

### PR DESCRIPTION
**Pull request by littleGnAl**
Automated changes by [create-pull-request](https://github.com/peter-evans/create-pull-request) GitHub action

### CODE REVIEW

File: lib/src/agora_base.dart [hunk 1]
```diff
@@ -560,7 +560,7 @@ enum InterfaceIdType {
   @JsonValue(7)
   agoraIidRtcConnection,
 
-  /// @nodoc
+  /// This interface class is deprecated.
```
Comment (maxxfrazer) [inline lib/src/agora_base.dart:563]: @Deprecated annotation should be used for this.

File: lib/src/agora_media_player.dart [hunk 2]
```diff
@@ -11,7 +11,7 @@ abstract class MediaPlayer {
   int getMediaPlayerId();
 
   /// Opens the media resource.
-  /// This method is called asynchronously. If you need to play a media file, make sure you receive the onPlayerSourceStateChanged callback reporting playerStateOpenCompleted before calling the play method to play the file.
+  /// This method is called asynchronously.If you need to play a media file, make sure you receive the onPlayerSourceStateChanged callback reporting playerStateOpenCompleted before calling the play method to play the file.
```
Comment (maxxfrazer) [inline lib/src/agora_media_player.dart:14]: This space should not have been deleted

File: lib/src/agora_media_player.dart [hunk 3]
```diff
@@ -201,7 +201,7 @@ abstract class MediaPlayer {
...
...eive the onPlayerSourceStateChanged callback reporting playerStateOpenCompleted before calling the play method to play the file.After you call this method, Agora opens the media resources and tries to obtain all the CDN routes for playing the media resource. By default, Agora uses the first CDN route for playing, and you can call the switchAgoraCDNLineByIndex method to switch routes.If you want to ensure the security of the connection and media files, to determine the sign and the ts fields for authentication. Once the fields are determined, use them as the query parameter of the URL to update the URL of the media resource. For example:The URL of the media file to be opened: rtmp://$domain/$appName/$streamNameThe URL updated by the authentication of the media file to be opened: rtmp://$domain/$appName/$streamName?ts=$ts&sign=$signAuthentication information:sign: An encrypted string calculated according to the MD5 algorithm based on authKey, appName, streamName, and ts. You need to for your authKey.ts: The timestamp when the authentication information expires. You can set the validity period of the authentication information according to your scenarios. For example, 24h or 1h30m20s.
```
Comment (maxxfrazer) [inline lib/src/agora_media_player.dart:204]: This space should not have been deleted

File: lib/src/agora_rtc_engine.dart [hunk 4]
```diff
@@ -1286,7 +1286,7 @@ class ChannelMediaOptions {
   @JsonKey(name: 'publishMediaPlayerId')
   final int? publishMediaPlayerId;
 
-  /// The user role. See ClientRoleType for details.
+  /// The user role. See ClientRoleType .
```
Comment (maxxfrazer) [inline lib/src/agora_rtc_engine.dart:1289]: Should "for details" be removed? If so, the trailing space should also.

File: lib/src/agora_rtc_engine.dart [hunk 5]
```diff
@@ -2723,19 +2722,19 @@ class DirectCdnStreamingMediaOptions {
 /// The basic interface of the Agora SDK that implements the core functions of real-time communication.
 /// RtcEngine provides the main methods that your app can call.
 abstract class RtcEngine {
-  /// Releases the RtcEngine instance.
+  /// Releases the instanceRtcEngine.
```
Comment (maxxfrazer) [inline lib/src/agora_rtc_engine.dart:2725]: looks like a mistake

File: lib/src/agora_rtc_engine.dart [hunk 6]
```diff
@@ -3468,15 +3467,15 @@ abstract class RtcEngine {
   /// Sets the audio data format reported by onPlaybackAudioFrameBeforeMixing .
   ///
   ///
-  /// * [sampleRate] The sample rate (Hz) of the audio data, which can be set as 8000, 16000, 32000, 44100, or 48000.
+  /// * [null] The sample rate (Hz) of the audio data, which can be set as 8000, 16000, 32000, 44100, or 48000.
```
Comment (maxxfrazer) [inline lib/src/agora_rtc_engine.dart:3470]: probably shouldn't be null

**Review by maxxfrazer — CHANGES_REQUESTED** [review]
Most of these doc changes look incorrect.

**Comment by littleGnAl** [comment]
@maxxfrazer This is a test PR by the bot, I'll close it, sorry about that.

**Pull request closed without merging**